controllers/apartments_controllers.rb

```diff
@@ -0,0 +1,30 @@
+
+get '/apartments' do
+  @apartments =  Apartment.all
+  erb :index
+end
+
+get '/apartments/:id' do
+@id = params[:name]
```

Comment:
The thing in the `[:brackets]` has to match the symbol in the path. This should be `@id = params[:id]`.